work-scheduler-service/src/main/java/com/github/gaalgergo/workscheduler/service/employees/EmployeeServiceImpl.java

```diff
@@ -28,4 +29,10 @@
             .map(entity -> conversionService.convert(entity, Employee.class))
             .collect(Collectors.toList());
     }
+
+    @Override
+    public void save(Employee employee) {
```

Comment:
Make the `employee` parameter `final`.
Never push your code without running `mvn clean install` successfully first!